exporter/clickhousetracesexporter/migrations/000012_add_RPC_system_service_method.up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE signoz_traces.signoz_index_v2
```

Comment:
@makeavish  There is a requirement to show `Status Code` which can contain both HTTP Code or gRPC Code. Does it make sense to add a new column or just use the existing columns to achieve the same?

Reply:
We are already showing status code which has both HTTP code or RPC code on trace table. We are achieving that by just querying both of existing columns.

Reply:
I have added a new column since there are already two columns and adding a new column (and index) for each protocol/encoding (JSON rpc, thrift, etc..) doesn't give any edge over single column. And delegating the group by and aggregation such as count to ClickHouse than doing it for each column and then in-memory for final (`Status Code` filter) is also another overhead this gets rid of. What do you think?